kubevirt_scripts/mountcd.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
+
```

Comment:
This script does not have a lot of error handling, I wonder if you want to add `set -eu -o pipefail`

Reply:
Added.

Reply:
Added validations instead.

Reply:
leaving a bash script with no error handling is generally not a good idea
as suggested by Dmitry you should add at least `set -eu -o pipefail` 
I would also add -x to simplify debugging